### DISCUSSION TITLE
Import function

### DISCUSSION THREAD

**Discussion by Estarna** [Komet/MediaElch#1399] [Q&A] [answered]
Hi there,

how do the backup/export/import function work? I tried the export function. How can I re-import?

Thanks!

-Eric

**Comment by bugwelle** [comment, marked as answer]
Hi,

the "export" functionality is only for exporting to HTML or CSV. The import functionality is for MKV files that were scraped using MakeMKV.
MediaElch does not have a backup functionality. Because MediaElch works on NFO files, you should simply backup your media files (and NFO files). :-)

Regards,
Andre

**Reply by Estarna** [reply]
Thanks!
